sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -2342,18 +2342,22 @@ namespace detail {
 #ifndef __ESIMD_DWORD_BLOCK_2D_WIDTH_SCALE
 #define __ESIMD_DWORD_BLOCK_2D_WIDTH_SCALE (1)
 #endif
+
+enum class block_2d_op { prefetch, load, store };
```

Comment:
nit: should we explicitly set the base type to int8_t or something?

Reply:
Why, is there a BKM? It is in detail namespace.

Reply:
Sorry, I thought the underlying type being used was undefined (compiler specific) but I checked the spec and it's required to be int, nevermind.